openfisca_france_local/departements/haute_saone/aide_permis.py

```diff
@@ -0,0 +1,27 @@
+from openfisca_france.model.base import Variable, Individu, MONTH
+
+
+class haute_saone_aide_permis(Variable):
+    value_type = float
+    entity = Individu
+    definition_period = MONTH
+    label = "Montant de l'aide au permis de conduire pour le département de la Haute-Saone"
+    reference = 'https://www.haute-saone.fr/index.php/nos-e-services/aides-aux-familles/pack-jeunes/'
```

Comment:
Pour la discussion que l'on a eu concernant la date de publication de l'aide, j'ai pu retrouver la date du `29/12/2022` que tu as utilisé dans les paramètres en remontant les liens de la source déjà présente.

Je propose d'ajouter les liens qui permettent de retrouver l'info.
```suggestion
    reference = [
    'https://www.haute-saone.fr/index.php/nos-e-services/aides-aux-familles/pack-jeunes/',
    'https://www.mes-services-publics70.fr/e_services/mpb/',
    ]
```

Pour retrouver la date du 29 décembre 2020 il faut télécharger le PDF en cliquant sur `Arrêté d'homologation du service` sur la source que je propose d'ajouter